static/scripts/calendar.js

```diff
@@ -68,12 +69,13 @@ $(document).ready(() => {
 	const view = window.location.hash.substring(1);
 
 	const calendarLanguage = document.querySelector('html').getAttribute('lang') === 'de' ? deLocale : enLocale;
+	const calendarTimezone = document.querySelector('html').getAttribute('timezone');
 
 	const calendar = new Calendar(calendarElement, {
-		plugins: [dayGridPlugin, timeGridPlugin, interactionPlugin],
+		plugins: [dayGridPlugin, timeGridPlugin, interactionPlugin, momentTimezonePlugin],
 		defaultView: view || 'dayGridMonth',
 		editable: false,
-		timeZone: 'UTC',
+		timeZone: calendarTimezone || 'Europe/Berlin',
```

Comment:
why should it be Europe/Berlin here? No user timezone if nothing set?

Reply:
can we reuse the default value or better require that calendarTimezone exists?